### DISCUSSION TITLE
How to open a small pop-up screen within Flet application?

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#1235] [Q&A]
### Question

On a `Flet UI` page, while an activity is in progress, we wanted to display a detailed log in a small `pop-up` window screen. We tried using `alert dialog` to simulate `pop-up` screen, but this alert dialog is **not** moving on the page. it seems stuck or static. it is not movable. Hence, we don't want to use `alert dialog` for this. 

What is the other possible solution to bring up a small `popup screen` on Flet UI? or how to move `alertdialog` around on the page using `mouse`

Please suggest, thanks!

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by TheCodeCache** [comment]
Hi Team,
Request to please help with the suggestion. I searched through the existing set of questions but couldn't find the reference on the same. Went through the documentation as well, did not find any reference. or may be I am missing something here. please help

**Comment by azikgal** [comment]
To view the progress of the execution, you can add this
pb = ft.ProgressBar()
page.overlay.append(pb)
